build.sbt

```diff
@@ -26,32 +26,30 @@ resolvers ++= Seq(
   Resolver.typesafeRepo("releases"),
   Resolver.typesafeRepo("snapshots"),
   Resolver.sonatypeRepo("snapshots"),
-  "spray repo" at "http://repo.spray.io",
-  "jgit-repo" at "http://download.eclipse.org/jgit/maven"
+  ("spray repo" at "http://repo.spray.io").withAllowInsecureProtocol(true),
+  ("jgit-repo" at "http://download.eclipse.org/jgit/maven").withAllowInsecureProtocol(true)
 )
 
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.10")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.15.0")
 
-addSbtPlugin("io.spray" % "sbt-revolver" % "0.9.0")
+addSbtPlugin("io.spray" % "sbt-revolver" % "0.9.1")
 
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
-
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.8.2")
 
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
 
-addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.2.1")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.0")
 
 addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.9.0")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.5.2")
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.7.6")
 
 addSbtPlugin("com.tapad" % "sbt-docker-compose" % "1.0.35")
```

Comment:
this one is currently failing us in the CI - if it doesn't go through correctly we don't detect failure